package.json

```diff
@@ -35,6 +36,7 @@
   "devDependencies": {
     "@blackbaud/skyux": "2.38.0",
     "@blackbaud/skyux-builder": "1.30.0",
+    "@blackbaud/skyux-cli": "1.6.2",
```

Comment:
Added this so I can use `npx` without needing to installed globally. I'll add a carat so it'll allow a non-specific version.

Reply:
I'm thinking this should be `skyux-sdk/cli` eventually too? Alongside `skyux-sdk/builder` too, right?

Reply:
I'm confused by this addition. We don't recommend installing the CLI local to the project (none of our other libraries do this). Mind providing more explanation?

Reply:
Sure, the reason I'd had it added as a `devDependency` is to avoid installing the package globally on my personal machine and instead use `npx` to handle the commands. Totally fine with not including this, I've pulled it out